static/js/settings_account.js

```diff
@@ -178,68 +176,68 @@ exports.initialize_custom_date_type_fields = function (element_id) {
 };
 
 exports.initialize_custom_user_type_fields = function (element_id, user_id, is_editable,
-                                                       set_handler_on_update) {
-    const field_types = page_params.custom_profile_field_types;
-    const user_pills = new IntDict();
+                                                         set_handler_on_update) {
+        const field_types = page_params.custom_profile_field_types;
```

Comment:
You should probably disable your IDE's linter and then fix this spacing issue.

Reply:
Done.